packages/web-components/src/components/va-checkbox/va-checkbox.css

```diff
@@ -13,7 +13,11 @@
 }
 
 #error-message {
-    margin-bottom: 1.2rem;
+  margin-bottom: 1.2rem;
+}
+
+:host([error]:not([error=""])) label > span {
+  font-weight: normal;
```

Comment:
Since we removed the `font-weight: 700` declaration for the `:host([error]:not([error=''])) span` selector in the form-field-error CSS file, do we still need this?

Reply:
Removed.